entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+URI=https://api.github.com
+API_HEADER="Accept: application/vnd.github.v3+json"
+AUTH_HEADER="Authorization: token $GITHUB_TOKEN"
+git config --global --add safe.directory /github/workspace
+
+git remote set-url origin https://x-access-token:${!INPUT_PUSH_TOKEN}@github.com/$GITHUB_REPOSITORY.git
+git config --global user.name "$COMMIT_NAME"
+git config --global user.email "$COMMIT_EMAIL"
+
+git fetch origin $SOURCE_BRANCH
+git fetch origin $DESTINATION_BRANCH
+git checkout -b $DESTINATION_BRANCH origin/$DESTINATION_BRANCH
```

Comment:
differs slightly from our other github action as it only fetches the source branch rather than checking it out as well.
This change is reflected in the merge command below by merging `origin/$SOURCE_BRANCH`